### DISCUSSION TITLE
Pagination

### DISCUSSION THREAD

**Discussion by akheel147** [nick-keller/react-datasheet-grid#197] [Feature requests]
Hii firstly a great component. Is There any way to  manually add any kind of pagination to react datasheet grid because when am trying to do pagination the values are getting changed

**Comment by nick-keller** [comment]
Yes you could add pagination outside of the component, and you could only pass the current page to DSG.
But that would make the value change indeed.

I would suggest not doing pagination and having a single big page, much better UX.

**Reply by akheel147** [reply]
Thank you very much for the reply. Can I expect built in pagination any time soon from your side.
That would be really helpful for me. Again thanks...

**Reply by nick-keller** [reply]
No I am not planning on having pagination as a feature. The UX is imo not acceptable, it creates a lot of problems, and it does not solve any pain. Basically I see no valid reason to integrate it.
Of course you can still easily implement it on top of this component on your side tho ;)
